src/components/pages/LoginPage.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 
-function Loginpage() {
+function LoginPage() {
   // state  --------------
   const [inputvalue, setinputvalue] = useState("");
```

Comment:
```suggestion
  const [inputValue, setInputValue] = useState("");
```
la convention veut qu'on écrit le state en CamelCase 😉

Reply:
Oui c'est vrai, j'oublierais pas le CamelCase  prochainement :).